docs/README-develop-beaker-scripts.md

```diff
@@ -34,7 +34,11 @@ Install and set up the Puppet agent and `cisco_node_utils` gem as described in [
 
 ### Configure NX-OS
 
-You must enable the ssh feature and give sudo access to the 'devops' user for the Beaker workstation to access the Puppet agent during testing.
+You must enable ssh to allow the Beaker workstation to access the Puppet agent during testing.
+
+#### Enable SSH: bash-shell
+
+The ssh feature must be enabled and sudo access must be given to the 'devops' user.
```

Comment:
This implies that they must specifically have a `devops` userid. Consider something like this instead:

` ``
The beaker framework requires ssh access and a non-root userid with sudo privileges. For `bash-shell`, ssh access is enabled by configuring `feature ssh` and userids are created with the `username` configuration. The example below create a 'devops' userid with the role and shelltype settings needed for beaker. 

```